test-framework/sudo-compliance-tests/src/sudoedit.rs

```diff
@@ -0,0 +1,432 @@
+use sudo_test::{
+    helpers::assert_ls_output, Command, Env, EnvNoImplicit, TextFile, ETC_SUDOERS, ROOT_GROUP,
+};
+
+use crate::{Result, GROUPNAME, PANIC_EXIT_CODE, SUDOERS_ALL_ALL_NOPASSWD, USERNAME};
+
+#[cfg(not(target_os = "freebsd"))]
+const DEFAULT_EDITOR: &str = "/usr/bin/editor";
+#[cfg(target_os = "freebsd")]
+const DEFAULT_EDITOR: &str = "/usr/bin/vi";
```

Comment:
Maybe move these to sudo-compliance-tests/src/lib.rs to allow reusing them between visudo and sudoedit tests?